lib/views/watch_live/watch_live.dart

```diff
@@ -62,9 +58,15 @@ class LobbyScreen extends StatefulWidget {
 }
 
 class _LobbyScreenState extends State<LobbyScreen> {
-    //function to handle 'Leave Lobby' press
-  void leaveLobby(BuildContext context){
-    Navigator.pop(context); //pop current screen
+  /// Eventually, we'll connect with Firebase and Agora—
+  /// for now, it's set up to show the [ParticipantStreamScreen] after 5 seconds.
+  @override
+  void initState() {
+    super.initState();
+    Timer(
+      const Duration(seconds: 5),
+      () => navigator.pushReplacement(const ParticipantStreamScreen()),
+    );
   }
```

Comment:
Currently, there's no way to get to that awesome `ParticipantStreamScreen` you made, so I'm adding a `Timer` here that will automatically redirect after 5 seconds.